pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -330,7 +340,7 @@ def _doExecute(self):
             hubQ.send_message(self.__class__.__name__, _("Failed to save storage configuration..."))
             self.data.bootloader.bootDrive = ""
         else:
-            if self.autopart:
+            if self.autopart or (flags.automatedInstall and (self.data.autopart.autopart or self.data.partition.seen)):
```

Comment:
Please add a comment below this line that ``self.run()`` is actually "run the storage check". That would make things a lot clearer, I think.